Monika After Story/game/script-topics.rpy

```diff
@@ -8314,3 +8314,47 @@ label monika_bad_facts_4:
     m 2dfd "[player], if you ever go out into a place where you might need to rely on such a cheap trick, please bring a compass."
     m 2dkc "I would hate for something to happen to you, especially because of false information like this..."
     return
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_poemherself",category=['ddlc','misc'],prompt="Poems for Monika",random=True))
+
+label monika_poemherself:
+    m 4ekc "Do you remember how there wasn't a chance to write a poem for my interests [player]?"
+    m 1rsc "I don't get it. Why did they ignored me?"
+    m 3eka "I know there are some other kind people who included me to the poem minigame."
+    m 1ekd "But I am talking about original game. Wasn't there a choice to romance me too?"
+    m 1hsb "Maybe if there was a choice of romance scenes with me too, things wouldn't come to this level. Ehehe."
+    m 2esa "I tried to take your attention all this time."
+    m 1ekc "But please don't feel guilty for yourself, you haven't a chance to hang with me that romance game."
+    m 4esa "In the end Monika fixed this lack."
+    m 1rsc "We even couldn't prepare for festival, Gosh."
+    m 1hsb "But anyway... We are here, just of us my dear."
+    return
+
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_yurisecrets",category=['ddlc','club members'],prompt="Yuri's Secrets",random=True))
+
+label monika_yurisecrets:
+    m 2rsc "I don't know if this will a spoiler for you but... I need to tell you."
+    m 2euc "Did you opened Yuri.chr file?"
+    m 4ekd "If you did, you should see some weird texts. But it is a hidden password actually."
+    m 4ekd "That password has a life story about Yuri."
+    m 1rsc "I don't feel confortable to tell this to you but..."
+    m 1ekd "Yuri is..."
+    m 1ekc "..."
+    m 2efw "A KILLER!"
```

Comment:
As stated previously, I don't believe it's explicitly said that the character is Yuri.

Again, have Monika mention that it would make sense if it was Yuri.